Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -458,6 +463,20 @@ public sealed partial class HumanoidProfileEditor : Control
 
             #endregion Markings
 
+            #region FlavorText
+
+            if (_configurationManager.GetCVar<bool>("ic.flavor_text"))
```

Comment:
```suggestion
            if (_configurationManager.GetCVar(CCVars.FlavorText))
```

dont access them by name